app/src/main/AndroidManifest.xml

```diff
@@ -31,7 +31,8 @@
         tools:targetApi="33">
         <activity
             android:name=".MainActivity"
-            android:configChanges="orientation|screenSize|screenLayout|keyboardHidden"
+            android:theme="@style/Theme.ElementX.Splash"
+            android:configChanges="orientation|screenSize|screenLayout|keyboardHidden|uiMode"
```

Comment:
`uiMode` is added. OK. I am surprised to see that change. What about language change? Will it crash the app?

Reply:
The crash is fixed for any cases (state and process restoration).
It's just that compose already takes care of uiMode changes, so no need for a configChange.